velox/exec/tests/utils/AssertQueryBuilder.h

```diff
@@ -175,6 +175,9 @@ class AssertQueryBuilder {
       memory::MemoryPool* pool,
       std::shared_ptr<Task>& task);
 
+  /// Run the query and return the result number.
+  vector_size_t AssertQueryBuilder::countResults(std::shared_ptr<Task>& task);
```

Comment:
This needs to be int64_t.

vector_size_t is int32_t which is enough for vector size, but not enough for the sum across many vectors.